CONFIG.md

```diff
@@ -0,0 +1,25 @@
+# Pylabrad Configuration
+
+There are a number of environment variables that can be used to configure
+labrad, which pylabrad also checks. These are as follows:
+
+- `LABRADHOST`: The hostname of the machine where the labrad manager is running
+  (default = localhost).
+- `LABRADPORT`: The tcp port on which the labrad manager is listening
+  (default = 7682).
+- `LABRADPASSWORD`: The password to use when connecting to labrad.
+- `LABRAD_TLS`: The transport-layer security (TLS) mode to use. Must be one of
+  the following:
+  - `starttls`: The default; the initial connection is unencrypted and is then
+    upgraded using the STARTTLS mechanism to secure the connection.
+  - `on`: The initial connection is encrypted. In this mode the default port
+    will be `LABRAD_TLS_PORT` instead of `LABRADPORT`.
```

Comment:
I guess my thinking is that the ideal end goal would be the server listening on a single port in either "normal" STARTTLS mode (allowing unencrypted connections from localhost) or STARTTLS-FORCE mode (all connections must be TLS).  I can't forsee every wanting to ban unencrypted local connections -- it is too useful for debugging to be able to sniff the packets, but I can see that some people would want to.  I guess if you see the major use case being all connections have mandatory SSL, then the "ON" mode makes more sense.